### ISSUE TITLE
Bolder Font Rendering on macOS

### ISSUE THREAD

**Issue by laktak** [alacritty/alacritty#4033]
Linux: alacritty 0.4.3, i3
macOS: alacritty 0.5.0-rc1 (fa79758)

This is alacritty on arch:
<img width="1002" alt="alacritty-arch" src="https://user-images.githubusercontent.com/959858/87644987-27102480-c74d-11ea-9414-9268265c3f7c.png">

This is alacritty on macOS:
<img width="1002" alt="alacritty-mac" src="https://user-images.githubusercontent.com/959858/87644992-29727e80-c74d-11ea-9b17-caacee330b0c.png">

And iTerm
<img width="1002" alt="iterm2" src="https://user-images.githubusercontent.com/959858/87644995-2a0b1500-c74d-11ea-9d7b-d09feaf112f3.png">

iTerm and Alacritty on Arch match exactly but Alacritty on macOS is slightly bolder (with `use_thin_stroke` set to `true`).

Mac:
```
[2020-07-25 21:30:20.555400000] [DEBUG] Estimated DPR: 2
[2020-07-25 21:30:20.555455000] [DEBUG] Estimated Cell Size: 18 x 38
[2020-07-25 21:30:20.555474000] [DEBUG] Estimated Dimensions: None
[2020-07-25 21:30:20.625237000] [INFO] Device pixel ratio: 2
[2020-07-25 21:30:20.635143000] [INFO] Initializing glyph cache...
[2020-07-25 21:30:20.671968000] [INFO] ... finished initializing glyph cache in 0.036756098s
[2020-07-25 21:30:20.672030000] [INFO] Cell Size: 18 x 38
[2020-07-25 21:30:20.672052000] [INFO] Padding: 2 x 0
[2020-07-25 21:30:20.672072000] [INFO] Width: 1600, Height: 1200
[2020-07-25 21:30:20.673524000] [INFO] PTY dimensions: Line(31) x Column(88)
```

arch:
```
[2020-07-25 21:30:33.761483447] [DEBUG] Estimated DPR: 2.2916666666666665
[2020-07-25 21:30:33.762016573] [DEBUG] Estimated Cell Size: 18 x 39
[2020-07-25 21:30:33.762072055] [DEBUG] Estimated Dimensions: None
[2020-07-25 21:30:33.961535153] [INFO] Device pixel ratio: 2.2916666666666665
[2020-07-25 21:30:33.971870876] [INFO] Initializing glyph cache...
[2020-07-25 21:30:34.002103695] [INFO] ... finished initializing glyph cache in 0.030186521s
[2020-07-25 21:30:34.002159069] [INFO] Cell Size: 18 x 39
[2020-07-25 21:30:34.002168378] [INFO] Padding: 4 x 0
[2020-07-25 21:30:34.002178083] [INFO] Width: 800, Height: 600
[2020-07-25 21:30:34.020879650] [INFO] PTY Dimensions: Line(15) x Column(44)
[2020-07-25 21:30:34.031096791] [INFO] Initialisation complete
[2020-07-25 21:30:34.088548586] [DEBUG] [unhandled] esc_dispatch ints=[], byte='k' (6b)
[2020-07-25 21:30:34.208797940] [DEBUG] New num_cols is 170 and num_lines is 46
[2020-07-25 21:30:34.209257300] [INFO] Width: 3072, Height: 1800
```

**Comment by chrisduerr** [comment]
> Alacritty on macOS is slightly bolder (with use_thin_stroke set to true).

Have you tried setting it to `false` and restarting Alacritty?

**Comment by laktak** [comment]
Yes, with `false` the font is even bolder. Should I attach a screenshot?

The font is "Consolas".

**Comment by chrisduerr** [comment]
Can you reproduce this on previous versions, like 0.4.2?

**Comment by ananthakumaran** [comment]
Non Retina
![Screenshot 2020-07-26 at 10 04 59 AM](https://user-images.githubusercontent.com/149238/88471558-677b5980-cf28-11ea-87f2-454dec1abeb7.png)
Retina
<img width="1614" alt="Screenshot 2020-07-26 at 10 05 39 AM" src="https://user-images.githubusercontent.com/149238/88471562-6a764a00-cf28-11ea-895b-1b5d1955fae0.png">

I tried with 4.3 and 4.2, the issue happens on both versions. Changing `use_thin_strokes` doesn't seem to have effect. Interestingly, thes issue goes away when I move the windows to retina screen.

**Comment by ivan-volnov** [comment]
Yeh, I have the same situation:
![Screenshot 2020-08-13 at 6 04 31 PM](https://user-images.githubusercontent.com/51086293/90122254-93ebfe00-dd8f-11ea-8e57-b5ce381c76d3.png)

Alacritty 0.5.0 on the left and iTerm2 on the right on non-retina display.
The font is a way bolder and contains some artefacts like cropped on the top "o" letter and cropped tilda also on the top.

The font is SF Mono Regular.

May be it's related to #3756 ?

**Comment by sluukkonen** [comment]
I've noticed that as of 0.50.0, Alacritty's font rendering is a lot sharper.

Here's Alacritty and Terminal.app side-by-side, both using Cascadia Code PL at 14 px. `use_thin_strokes` is set to false with Alacritty.

![Screenshot 2020-09-10 at 12 34 24](https://user-images.githubusercontent.com/39655/92712296-1dc4c200-f362-11ea-915f-937c8696a1a8.png)

![Screenshot 2020-09-10 at 12 34 17](https://user-images.githubusercontent.com/39655/92712319-23220c80-f362-11ea-8df0-af96a10c05be.png)

(Forgive me, for the terminal colors don't match)

**Comment by ivan-volnov** [comment]
Any news here?

**Comment by VKondakoff** [comment]
I see this on Retina screen as well (left: Alacritty 0.9.0, right: iTerm2, macOS Monterey):
<img width="416" alt="Снимок экрана 2021-11-01 в 15 13 54" src="https://user-images.githubusercontent.com/230824/139746585-1b30905d-d2c9-4691-ab27-43d7d7b5a6f7.png">

**Comment by VKondakoff** [comment]
Is [this issue](https://github.com/alacritty/alacritty/issues/3804) somehow related?

**Comment by chrisduerr** [comment]
It likely is. The macOS thin stroke behavior has changed so much in recent years I've honestly just given up on keeping track.

**Comment by ofttryaj** [comment]
there is also issue on MacOS 12

**Comment by applebait** [comment]
> It likely is. The macOS thin stroke behavior has changed so much in recent years I've honestly just given up on keeping track.
Oh, of course, let's blame Apple and macOS! Ahahaha! If so why other Terminal emulators render it properly?

**Comment by Sciencentistguy** [comment]
I am also having this issue, on macos 12.3.1 and alacritty 0.10.1.

Alacritty is left, iTerm2 is right:

![image](https://user-images.githubusercontent.com/4983935/176029947-00513abd-40af-43ae-885e-6c15722dd6a4.png)

**Comment by ecogest** [comment]
Same issue on macOS 12.4, Alacritty 0.10.1.

**Comment by chrisnc** [comment]
I think this is fixed here: https://github.com/alacritty/alacritty/commit/2a676dfad837d1784ed0911d314bc263804ef4ef

**Comment by gszr** [comment]
Was facing the same behavior in Ventura, can confirm this fixes it! 😃 

> I think this is fixed here: [2a676df](https://github.com/alacritty/alacritty/commit/2a676dfad837d1784ed0911d314bc263804ef4ef)
